docs/installing_hypershift_cli.md

```diff
@@ -6,23 +6,24 @@ The Hosted Control Plane command-line interface (CLI) is an `oc` command-line [p
 
 ## Installing the Hosted Control Plane CLI
 
-**NOTE: This document is unofficial will be updated later. For now download the CLI binary from this page
+1. [Enable the Hypershift feature](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/provision_hosted_cluster_on_mce_local_cluster.md#configuring-the-hosting-service-cluster)
 
-1. Download the HyperShift CLI binary.
+2. On the OCP console, click on the ? button at the top right hand corner and choose `Command line tools` menu to download the HyperShift CLI binary.
 
+(unofficial: you can download one of these as well)
 - [HyperShift CLI binary for Linux x86_64](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/hypershift-linux-amd64.tar.gz?raw=true)
 - [HyperShift CLI binary for Mac x86_64](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/hypershift-darwin-amd64.tar.gz?raw=true)
 - [HyperShift CLI binary for Windows x86_64](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/hypershift-windows-amd64.tar.gz?raw=true)
```

Comment:
Going to poke at how we call it HyperShift CLI here, and then talk about hcp below.

`Hypershift hosted control plane CLI`